TabnineSelfHostedForMarketplace/src/main/java/com/tabnineSelfHosted/statusBar/SelfHostedForMarketplaceStatusBarActions.kt

```diff
@@ -0,0 +1,30 @@
+package com.tabnineSelfHosted.statusBar
+
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.DefaultActionGroup
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.options.ShowSettingsUtil
+import com.intellij.openapi.project.DumbAwareAction
+import com.intellij.openapi.project.Project
+import com.tabnineSelfHosted.userSettings.AppSettingsConfigurable
+
+object SelfHostedForMarketplaceStatusBarActions {
```

Comment:
```suggestion
object TabnineSelfHostedForMarketplaceStatusBarActions {
```
just to prevent collisions with other plugins who register a class with a similar name (slim chance, but it did happened a few time before so just to be on the safe side)